src/treemap.ts

```diff
@@ -273,8 +274,41 @@ export class TreeMap {
       let address = getAddress(node);
       this.zoom(address);
     };
+    dom.onkeydown = e => {
+      if (!(e.target instanceof HTMLElement)) return;
+
+      let elem;
+      switch (e.key) {
+        // zoom to selection
+        case 'Enter':
+          const address = getAddress(e.target);
+          this.zoom(address);
+          return;
+        // move selection focus
+        case 'ArrowUp':
+          elem = e.target.parentElement;
+          break;
+        case 'ArrowDown':
+          elem = e.target.querySelector('.webtreemap-node');
+          break;
+        case 'ArrowLeft':
+          elem = e.target.previousElementSibling;
+          if (elem && !isDOMNode(elem)) {
+            elem = e.target.parentElement;
```

Comment:
I know it's a kinda common pattern to have ArrowLeft eventually act like ArrowUp at the leftmost boundary, but in this particular case I found it confusing since the children tile multiple rows and I would accidentally go too far left and end up above where I wanted to be.

So I think this should just only navigate between children, especially since ArrowRight has no special behavior.